histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -3,6 +3,7 @@
 import os
 import time
 
+import dask
```

Comment:
One of the reasons this wasn't imported at the top level is that it is a heavy import (that is, it takes time), so users of the library that aren't using dask end up with a slower start.  It should be moved back into the function where it gets used.

Reply:
The import dask is moved to `detect_nuclei_with_dask` function. After the change, it is tested on some sample images to make sure it is working. On the first run it is taking some time to load but from the second time the execution is faster.